Server/src/main/java/local/AssetTypeDAOImpl.java

```diff
@@ -58,6 +61,20 @@ public String getAssetTypeBoundary(String asset_type_id, String boundary_type){
         return boundary;
     }
 
+    public HashMap getAssetTypeBoundaries(String asset_type_id) {
```

Comment:
Add the @override tag so that it shows in the interface that this class extends.

Reply:
Resolved in the last commit,